registry/zookeeper/registry.go

```diff
@@ -134,6 +134,7 @@ func (r *zkRegistry) InitListeners() {
 					regConfigListener.Close()
 				}
 				newDataListener.SubscribeURL(conf, NewRegistryConfigurationListener(r.client, r))
+				r.WaitGroup().Add(1)
 				go r.listener.ListenServiceEvent(conf, fmt.Sprintf("/dubbo/%s/"+constant.DEFAULT_CATEGORY, url.QueryEscape(conf.Service())), newDataListener)
```

Comment:
I think the operation which is called add wait group should relate to listen service event.
So they can extract a new method.

Reply:
Waitgroup is only used to record the number of goruntines. It is not intended to be bound to the listenserviceevent method

Reply:
Waitgroup is only used to record the number of goruntines. It is not intended to be bound to the listenserviceevent method